internal/config/init/config.go

```diff
@@ -0,0 +1,52 @@
+package init
+
+import (
+	"github.com/spf13/viper"
+	"github.com/wal-g/wal-g/internal"
+	"github.com/wal-g/wal-g/internal/config"
+	"github.com/wal-g/wal-g/internal/tracelog"
+)
+
+func init() {
+	for _, adapter := range internal.StorageAdapters {
+		allowedConfigKeys := []string{
```

Comment:
Why you do reinitialize allowedConfigKeys?

Reply:
My bad. Fixed.